modular_skyrat/modules/customization/modules/mob/living/carbon/human/human_update_icons.dm

```diff
@@ -233,7 +233,12 @@
 		if(is_taur && (BP.body_part == LEG_LEFT || BP.body_part == LEG_RIGHT))
 			continue
 
-		new_limbs += BP.get_limb_icon()
+		var/list/image/limb_icon = BP.get_limb_icon()
+		if (dna && dna.species.specific_alpha)
+			for (var/x in limb_icon)
+				var/image/IX = x
+				IX.alpha = dna.species.specific_alpha
```

Comment:
If you're setting the bodypart alpha, you can just use it without checking species again.